tests/bin/check/run.t

```diff
@@ -41,13 +51,16 @@ If the condition described above is fulfilled, there are 4 checks to be performe
   [-] Performing lint for package my_pkg in <test_directory>
   [FAIL] File README is missing.
   [FAIL] File LICENSE is missing.
-  [FAIL] File CHANGES is missing.
+  [ OK ] File CHANGES is present.
```

Comment:
Please keep one test where the changelog is missing, so this test case doesn't fall by the wayside.

Reply:
Fixed